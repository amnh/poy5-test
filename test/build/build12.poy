(* Build with iterative *)
(*terminals reduced to 8 to expidite test*)
read ("2.fas")
set (seed:50)
set (iterative:true)
build (1)
report (treestats)
