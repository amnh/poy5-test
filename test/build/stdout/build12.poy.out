Trees Found:
                                            
Tree length    Number of hits
334.           1

